promql/prettier/prettier_test.go

```diff
@@ -0,0 +1,446 @@
+package prettier
+
+import (
+	"testing"
+
+	"github.com/prometheus/prometheus/util/testutil"
+)
+
+type prettierTest struct {
+	expr     string
+	expected string
+}
+
+var sortingLexItemCases = []prettierTest{
+	{
+		expr:     `sum(go_alloc_bytes) by (job)`,
+		expected: `sum by (job) (go_alloc_bytes)`,
+	},
+	{
+		expr:     `sum by (job) (go_alloc_bytes)`,
+		expected: `sum by(job) (go_alloc_bytes)`,
+	},
+	{
+		expr:     `metric_first + metric_second`,
+		expected: `metric_first + metric_second`,
```

Comment:
This is a vector matching and should be split over multiple lines.

Reply:
This is addressed now. Resolving.